components/badge/badge.css

```diff
@@ -53,7 +50,22 @@
   background: var(--clr-grey-80);
```

Comment:
just a suggestion naming variables in caps would improve readability. Everything else looks good

Reply:
I understand, but it depends on the preferences of the developer. For readability, I make sure to write "clr" followed by the color name, which is followed by a number to let another developer know how dark or light the hue is. I even asked Animesh about this and he approves. Thank you so much for reviewing @vrx29.